src/components/LiveChart/index.tsx

```diff
@@ -0,0 +1,243 @@
+        </Flex>
+      ) : (
+        <>
+          <Flex justifyContent="space-between" alignItems="center">
+            <Flex>
+              <DoubleCurrencyLogo
+                currency0={nativeInputCurrency}
+                currency1={nativeOutputCurrency}
+                size={24}
+                margin={true}
+              />
+              <Flex alignItems="center" fontSize={isMobile ? 14 : 18} color={theme.subText}>
+                <span style={{ marginRight: '8px' }}>
+                  <Text fontSize={isMobile ? 18 : 24} fontWeight={500} color={theme.text} display="inline-block">
+                    {nativeInputCurrency?.symbol}
+                  </Text>
+                  <span style={{ whiteSpace: 'nowrap' }}>{' / ' + nativeOutputCurrency?.symbol}</span>
+                </span>
+                <SwitchButtonWrapper onClick={onRotateClick}>
+                  <Repeat size={14} />
+                </SwitchButtonWrapper>
+              </Flex>
+            </Flex>
+            {isMobile && renderTimeframes()}
+          </Flex>
+          <Flex justifyContent="space-between" alignItems="flex-start" marginTop={'5px'}>
+            <Flex flexDirection="column" alignItems="flex-start">
+              {showingValue === 0 || error ? (
+                <Text fontSize={28} color={theme.subText}>
+                  --
+                </Text>
+              ) : (
+                <AnimatingNumber
+                  value={showingValue}
+                  symbol={nativeOutputCurrency?.symbol}
+                  fontSize={isMobile ? 24 : 28}
+                />
+              )}
+              <Flex marginTop="2px">
+                {showingValue === 0 || error ? (
+                  <Text fontSize={12} color={theme.disableText}>
+                    --
+                  </Text>
+                ) : (
+                  <>
+                    <Text fontSize={12} color={different >= 0 ? '#31CB9E' : '#FF537B'} marginRight="5px">
+                      {different} ({differentPercent}%)
+                    </Text>
+                    <Text fontSize={12} color={theme.disableText}>
+                      {getTimeFrameText(timeFrame)}
+                    </Text>
+                  </>
+                )}
+              </Flex>
+            </Flex>
+            {!isMobile && renderTimeframes()}
+          </Flex>
+          <div style={{ flex: 1, marginTop: '12px' }}>
+            {loading || error ? (
+              <Flex
+                minHeight={isMobile ? '240px' : '292px'}
+                flexDirection={'column'}
+                alignItems={'center'}
+                justifyContent={'center'}
+                color={theme.disableText}
+                style={{ gap: '16px' }}
+              >
+                {loading && <Loader />}
+                {error && (
+                  <>
+                    <WarningIcon />
+                    <Text fontSize={16}>There was an error with the chart. Try again later</Text>
```

Comment:
Please wrap an `<Trans></Trans>` tag for multi language functionality

Reply:
I fixed all the feedbacks, please recheck, thanks